Sources/Private/CoreAnimation/Extensions/Keyframes+combined.swift

```diff
@@ -0,0 +1,222 @@
+// Created by Cal Stephens on 1/28/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+// MARK: - Keyframes
+
+enum Keyframes {
+
+  // MARK: Internal
+
+  /// Combines the given keyframe groups of `Keyframe<T>`s into a single keyframe group of of `Keyframe<[T]>`s
+  ///  - If all of the `KeyframeGroup`s have the exact same animation timing, the keyframes are merged
+  ///  - Otherwise, the keyframes are manually interpolated at each frame in the animation
+  static func combined<T>(_ allGroups: [KeyframeGroup<T>]) -> KeyframeGroup<[T]>
```

Comment:
These methods were renamed from `Keyframes.combinedIfPossible(...) -> KeyframeGroup?` to `Keyframes.combined(...) -> KeyframeGroup`